.gitignore

```diff
@@ -5,3 +5,4 @@ website/error_log
 # Ignore directories generated by Composer
 vendor/
 composer.lock
+.idea/
```

Comment:
This should not be added to the repo but should be [ignored globally](https://stackoverflow.com/questions/59328900/how-to-globally-ignore-the-idea-folder-in-git-not-just-adding-it-to-gitignor).